services/core-web/common/selectors/externalAuthorizationSelectors.js

```diff
@@ -0,0 +1,3 @@
+import * as externalAuthReducer from "../reducers/externalAuthorizationReducer";
+/* eslint-disable import/prefer-default-export */
+export const getMineEpicInfo = (state) => externalAuthReducer.getMineEpicInfo(state);
```

Comment:
Can you update to use the newer export syntax? 

```
export const {getMineEpicInfo} = externalAuthReducer
```

Reply:
Done